src/components/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+import './Header.css';
+
+export const Header = () => {
+  return (
+    <header>
+      <Link className='' to={'/'}>
+        <img
+          className='headerLogo'
+          src='https://github.com/Netflix4-0/Netflix4.0/assets/117076586/4c41d512-ea91-4829-909f-5d0d4dfa52a3'
+          alt='logo'
+        />
+      </Link>
+      <Link className='link' to={'/categories'}>
+        Categories
+      </Link>
+      <Link className='link' to={'/'}>
+        Bookmarks
```

Comment:
The bookmarkPage is implemented although not yet styled, I suggest implementing the path right away

Reply:
Yeah defenetly! Dont think that page existed when I made this PR, but now its added and I merged main into the branch again so now we should be ready to roll! :)